DragonSlayer.h

```diff
@@ -1,8 +1,18 @@
 #pragma once
+#include "Character.h"
 
 struct DragonSlayer : Character
 {
+    DragonSlayer(const std::string& name_, int hp, int armor_);
+
+    const std::string& getName() override;
+    std::string getStats() override;
+
     void attack(Character& other) override;
+
+protected:
+    std::vector<std::unique_ptr<Item>> attackItems;
+
```

Comment:
No one is inheriting from DragonSlayer. 
There is no reason to make this variable `protected`. 
it should be `private`
- [ ] revise